docs/gaia/gaiacli.md

```diff
@@ -761,3 +761,26 @@ The transaction can now be sent to the node:
 ```bash
 gaiacli tx broadcast signedTx.json
 ```
+
+## Shells completion scripts
+
+Completion scripts for popular UNIX shell interpreters such as `Bash` and `Zsh`
+can be generated through the `completion` command, which is available for both
+`gaiad` and `gaiacli`.
+
+If you want to generate `Bash` completion scripts run the following command:
+
+```bash
+gaiad completion > gaiad_completion
+gaiacli completion > gaiacli_completion
+```
+
+If you want to generate `Zsh` completion scripts run the following command:
+
+```bash
+gaiad completion --zsh > gaiad_completion
+gaiacli completion --zsh > gaiacli_completion
+```
+
+Refer to the user's manual of your interpreter for information
```

Comment:
While this is admirable, maybe we provide short examples of how to link it in your `bash` and `zsh` profiles?

Reply:
Maybe just link to docs: https://debian-administration.org/article/316/An_introduction_to_bash_completion_part_1
https://github.com/spf13/cobra/blob/master/bash_completions.md

Reply:
more info here: https://kubernetes.io/docs/tasks/tools/install-kubectl/#enabling-shell-autocompletion